mmv1/products/osconfig/api.yaml

```diff
@@ -380,10 +380,6 @@ objects:
             properties:
              - !ruby/object:Api::Type::Array
               name: 'classifications'
-              exactly_one_of:
```

Comment:
Do we need at least_one_of for these three fields, as the previous logic was exactly_one_of, but now since we remove the restriction, it not only allows three fields being set at the same time but also allow none of them being set in the config? Would you mind verifying if that's valid? Thanks!